### DISCUSSION TITLE
is there a new release planned or a release schedule?

### DISCUSSION THREAD

**Discussion by mbakker7** [modflowpy/flopy#1808] [General]
Dear FloPy Developers,

Have you planned the next official FloPy release? The last one was in December in 2022. 
Do you maybe even have a release schedule? 
I would like to use some of the latest fixes in a workshop and I am not sure I should teach the students to install the dev version rather than the official release. 

Thanks in advance!

**Comment by aleaf** [comment]
Along these lines, I think there might be value in trying to do minor and patch releases more frequently. As things are now, LOTs of changes fly by between patch releases, including important bug fixes and even interface changes that can break code. This can make it more difficult to debug issues with less experienced users who are either working off the last pip install or some intermediate commit on the develop branch. I'd imagine more frequent releases might be helpful to anyone wanting to use the plugins as well.

**Reply by mbakker7** [reply]
Following up on that, are there any examples of the plugins? They sound cool.

**Reply by christianlangevin** [reply]
FloPy plugins, refactored from [this PR](https://github.com/modflowpy/flopy/pull/1746), will be coming soon to [flopy-api ](https://github.com/modflowpy/flopy-api)and [flopy-api-examples](https://github.com/modflowpy/flopy-api-examples) soon.  Stay tuned.

**Comment by christianlangevin** [comment]
Hey @mbakker7, we are working to have both flopy and MODFLOW 6 releases this month.  I think it would be great to have minor and patch releases more frequently.  That is something we are working toward.
